cmd/build/build.go

```diff
@@ -14,6 +14,7 @@ import (
 	"github.com/styrainc/opa-control-plane/cmd/internal/flags"
 	"github.com/styrainc/opa-control-plane/internal/config"
 	"github.com/styrainc/opa-control-plane/internal/logging"
+	"github.com/styrainc/opa-control-plane/internal/migrations"
```

Comment:
I can't believe I just noticed this, but apparently no one has adjusted the `go.mod` to be `github.com/open-policy-agent/opa-control-plane`, instead of the original `github.com/styrainc` prefix. 🤦

Reply:
I'll file a task issue for that-- it doesn't need to be fixed in this PR, since it's a logically distinct change.

Reply:
https://github.com/open-policy-agent/opa-control-plane/issues/119